src/integrationTest/resources/db/migration/V1_9999__integrationTestData.sql

```diff
@@ -235,3 +235,10 @@ VALUES (-2, -2, -3, 'OPEN', 'DOWNLOAD', TIMESTAMP WITH TIME ZONE '2023-06-26 14:
 
 INSERT INTO transient_object_directory (tod_id, moj_mer_id, moj_ods_id, external_location, checksum, created_ts, modified_ts)
 VALUES (-1, -1, 1, 'f744a74f-83c0-47e4-8bb2-2fd4d2b68647', NULL, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+
+INSERT INTO moj_media (moj_med_id, moj_ctr_id, c_channel, c_total_channels, c_start, c_end, i_version)
+VALUES (-1, 1, 1, 4, TIMESTAMP WITH TIME ZONE '2023-07-03 17:00:00+00:00', TIMESTAMP WITH TIME ZONE '2023-07-03 17:30:00+00:00', 0);
+
+INSERT INTO external_object_directory (eod_id, moj_med_id, moj_ods_id, external_location, external_location_type, created_ts, modified_ts)
+VALUES(-1, -1, 1, 'a7ad5828-6a20-4bd0-adb1-bf1496a2622a', 'unstructured', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
```

Comment:
There was some discussion on the approach of using Flyway for populating int test data here: https://hmcts-reform.slack.com/archives/C04T0CVJ28N/p1688381752033339

Not sure if you want to address this here by populating the database via entities in the test setup, or leave this for another time.

Reply:
I've left the `shouldGetMediaLocation` test here to use the SQL inserts in the `src/integrationTest/resources/db/migration/V1_9999__integrationTestData.sql` script using the `moj_med_id=-1` and updated the `moj_ods_id=2`